packages/app-store/routing-forms/trpc/formMutation.handler.ts

```diff
@@ -24,58 +24,6 @@ interface FormMutationHandlerOptions {
   input: TFormMutationInputSchema;
 }
 
-function throwIfInvalidQueryValueToBeSaved({
-  routes,
-}: {
-  routes: FormMutationHandlerOptions["input"]["routes"];
-}) {
-  if (!routes) {
-    return;
-  }
-  routes.forEach((route, routeIndex) => {
-    if (isRouter(route)) {
-      return;
-    }
-    // We use separate schema for queryValye here which is much more strict
-    // It allows that we are still lenient with schema while reading the queryValue but while saving it we are strict
-    const parsedFormFieldsQueryValue = queryValueSaveValidationSchema.safeParse(route.queryValue);
-    if (!parsedFormFieldsQueryValue.success) {
-      throw new TRPCError({
-        code: "BAD_REQUEST",
-        message: `Route ${routeIndex + 1} form fields: ${getErrorMessageFromZodError(
```

Comment:
Not needed now. As the default zod schema handles the same errors though they might not be that pretty